Spec/PushActivationStateMachine.swift

```diff
@@ -91,41 +105,70 @@ override class var defaultTestSuite : XCTestSuite {
+test__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__reusableTestsRsh3a2a(testCase: .the_local_device_has_id_and_deviceIdentityToken__emits_a_SyncRegistrationFailed_event_with_code_61002_if_client_IDs_don_t_match)
+}
+
+func test__012__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__the_local_device_has_id_and_deviceIdentityToken__the_local_DeviceDetails_matches_the_instance_s_client_ID__calls_registerCallback__transitions_to_WaitingForRegistrationSync() {
+test__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__reusableTestsRsh3a2a(testCase: .the_local_device_has_id_and_deviceIdentityToken__the_local_DeviceDetails_matches_the_instance_s_client_ID__calls_registerCallback__transitions_to_WaitingForRegistrationSync)
+}
+
+func test__013__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__the_local_device_has_id_and_deviceIdentityToken__the_local_DeviceDetails_matches_the_instance_s_client_ID__PUTs_device_registration__transitions_to_WaitingForRegistrationSync() {
+test__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__reusableTestsRsh3a2a(testCase: .the_local_device_has_id_and_deviceIdentityToken__the_local_DeviceDetails_matches_the_instance_s_client_ID__PUTs_device_registration__transitions_to_WaitingForRegistrationSync)
+}
+
 
                     // RSH3a2b
-                    context("local device") {
-                        it("should have a generated id") {
+                    
+                        func test__014__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__local_device__should_have_a_generated_id() {
+beforeEach__Activation_state_machine__State_NotActivated()
+
                             rest.internal.resetDeviceSingleton()
                             expect(rest.device.id.count) == 36
+afterEach()
+
                         }
-                        it("should have a generated secret") {
+                        func test__015__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__local_device__should_have_a_generated_secret() {
+beforeEach__Activation_state_machine__State_NotActivated()
+
                             guard let deviceSecret = rest.device.secret else {
                                 fail("Device Secret should be available because it's loaded when the getter of the property is called"); return
                             }
                             guard let data = Data(base64Encoded: deviceSecret) else {
                                 fail("Device Secret should be encoded with Base64"); return
                             }
-                            expect(data.count) == 32 //32 bytes digest
+                            expect(data.count) == 32 
+afterEach()
+//32 bytes digest
```

Comment:
TODO this comment has gotten messed up

Reply:
(Observation, doesn't block review)

The migrator has moved this comment to after the `afterEach` call for some reason. I'll see if I can figure out why, but if it's a huge amount of work then I'll just fix this manually (it seems to be the only such affected comment).